<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>330</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mahtab's Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 64);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="text_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(195, 195, 195);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 169, 255);</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 169, 255);</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tan">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 169, 255);</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(203, 169, 255);</string>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 196, 237);</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sqrt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 196, 237);</string>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_percent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 196, 237);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_negative">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 196, 237);</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 218, 255);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sub">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 218, 255);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mul">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 218, 255);</string>
    </property>
    <property name="text">
     <string>Ã—</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>40</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 218, 255);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_div">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 218, 255);</string>
    </property>
    <property name="text">
     <string>Ã·</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ac">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
